nodeJS/testing/Testing.md

```diff
@@ -9,7 +9,7 @@ By the end of this lesson, you should be able to do or answer the following:
 - Use the `supertest` module to test Express routes/controllers.
 - Describe how supertest handles our express application.
 - Explain the functionality `superagent` provides to supertest.
-- What is the purpose of `done`? What convenience does supertest provide concerning it?
+- Test asynchronous operations with use of the `done` parameter and supertest's methods.
```

Comment:
I'd like to see this reworded to be higher level. If you haven't yet read [this post on our wiki](https://github.com/TheOdinProject/curriculum/wiki/Learning-Outcomes-vs-Knowledge-Checks) it gives a broad example of what I mean.

If you'd like a specific example I can come up with one, just let me know, but I will leave it up to you:slightly_smiling_face:

Reply:
Do you mean something along the lines of 'Describe how the `done` parameter can be used to test asynchronous operations'?

Reply:
Probably more in the direction of "Describe what the `done` parameter is used for"